mobile/src/common/types/navigation/course-module-tab-item.type.ts

```diff
@@ -0,0 +1,8 @@
+import { CourseModuleScreenName } from '~/common/enums/enums';
+
+type TabCourseModuleNavigationItem = {
```

Comment:
Remove TabCourseModuleNavigationItem and InterviewTabItem types, use NavigationItem instead